connectors/src/connectors/github/temporal/workflows.ts

```diff
@@ -147,7 +144,6 @@ export async function githubReposSyncWorkflow(
             {
               dataSourceConfig,
               connectorId,
-              connectionId,
```

Comment:
@fontanierh this one is the only one where I have a doubt. Maybe I'll keep connectionId here just to be safe

Reply:
Yeah this won't be backwards compatible I think